tests/unit/.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
+  'extends': '../../.eslintrc',
+
+  'env': {
+    'mocha': true
+  },
+
+  'globals': {
+    'expect': false,
+    'sandbox': false,
+    'document': false,
+    'window': false,
+    'React': false,
+    'ReactDOM': false,
```

Comment:
I would like to import `React` and `ReactDOM` only once, not in every file. How can I make the linter not scream about it?